automatic/poi/poi.nuspec

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <!-- == PACKAGE SPECIFIC SECTION == -->
```

Comment:
You don't need these comments, however I'm not going to require you to remove them. Just an FYI

Reply:
Fixed in 80c1b3e